tools/driver/p4c_src/setup.cfg___for_pycodestyle

```diff
@@ -0,0 +1,23 @@
+[pycodestyle]
```

Comment:
Can you explain the purpose of this file? Is this a style file only for the driver code?

Reply:
Yup.